security.md

```diff
@@ -37,3 +37,90 @@ We are working to identify and coordinate security efforts across the Jupyter co
 The [Jupyter Security](https://github.com/jupyter/security) GitHub repo has information how to participate and contribute.
 For discussion, please use the special Discourse [security topic](https://discourse.jupyter.org/c/special-topics/security/48) on the Jupyter Discourse server.
 
+----
+
+Below is a copy of the vulnerability handling procedure that can be found on the [security repository](https://github.com/jupyter/security).
+
+# Jupyter Vulnerability Handling Process
+
+This document summarizes and proposes guidelines for handling vulnerabilities reported in Jupyter projects. 
+
+Security issues and vulnerabilities have expectations and processes that are differ from typical open source practices:
+ - Private discussions
+ - Obfuscation
+ - Short timeline
+
+This makes it quite hard to be able to understand, learn, or know what to expect from a security point of view. This document
+will give you a glimpse on what's happening on the inside, and what timeline to expect when you report a security vulnerability. 
+It will also serve as a guideline and task list for Jupyter Contributors and Developers on how to handle security relatied issues.
+
+## Scope
+
+This process applies to *all projects* governed by Jupyter, including JupyterLab, Jupyter Notebook, JupyterHub, and Jupyter Server.
```

Comment:
I would use the language in the governance model, where these are called "Subprojects" and possibly link to the Subprojects doc here: https://github.com/jupyter/governance/blob/master/list_of_subprojects.md